core/src/main/java/com/linecorp/armeria/internal/common/DefaultAttributeMap.java

```diff
@@ -44,11 +44,12 @@
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.Iterators;
 
+import com.linecorp.armeria.common.RequestContext;
 import com.linecorp.armeria.common.annotation.Nullable;
 
 import io.netty.util.AttributeKey;
 
-final class DefaultAttributeMap {
+public final class DefaultAttributeMap {
```

Comment:
nit; Might be worth adding `@UnstableApi` not that this class is `public`

Reply:
It's already in `internal` so `@UnstableAPI` is implied anyway.

Reply:
Seems so, feel free to ignore this comment 😅